deploy/README.md

```diff
@@ -0,0 +1,64 @@
+# Deploy on Google Cloud Platform
+
+If your dataset is private, there must be a Google Group of users who have
+access to the data. [Identity-Aware Proxy](https://cloud.google.com/iap/docs/)
+will be used to restrict Data Explorer to users in this Google Group.
+
+### Setup
+
+* [Follow these instructions](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery/deploy)
+to bring up Elasticsearch in GKE and index dataset.
+
+* If you haven't done so already, set the default project:
+
+  `gcloud config set project PROJECT_ID`
+
+* If you haven't done so already, create the App Engine application:
+
+  `gcloud app create`
+  
+### Enable Access Control
+
+* Follow the [instructions for setting up IAP](https://cloud.google.com/iap/docs/app-engine-quickstart#enabling_iap)
```

Comment:
I ran `gcloud app create` but in IAP page, there are no resources. Unfortunately there doesn't appear to be a way to undo `gcloud app create`, like `gcloud app delete`. To reproduce, you can just switch to a project where you haven't run `gcloud app create`. It doesn't need to a be an official Verily project.

How about this:
- Deploy the UI Server
- Enable Access Control (UI Server url only)
- Deploy the API Server

That should work, right?

Reply:
Done